packages/api-sdk/src/devices/client/__tests__/metriport.test.ts

```diff
@@ -7,7 +6,7 @@ jest.mock("axios");
 describe("getMetriportUserId", () => {
   const userId = "1";
   const apiKey = "FAKE_KEY";
-  const mockedAxios = mocked(axios, true);
+  const mockedAxios = axios as jest.Mocked<typeof axios>;
```

Comment:
Why not `mocked(axios...)`?

`as` only makes it to be treated as such, but it doesn't change its underlying implementation/code

Reply:
I was getting a typescript error in the build process that it didnt like unless i did something like this

Reply:
I was saying how the second param in the mocked function should have been an object even though its suppose to be a boolean

Reply:
@leite08

Reply:
Yea i removed boolean and it works now